gsy_e_sdk/setups/Asset_API_scripts/rest_bid_offer_posting.py

```diff
@@ -88,9 +88,12 @@ def on_finish(self, finish_info):
 if CONNECT_TO_ALL_ASSETS:
     registry = aggregator.get_configuration_registry()
     registered_assets = get_assets_name(registry)
-    LOAD_NAMES = registered_assets["Load"]
-    PV_NAMES = registered_assets["PV"]
-    STORAGE_NAMES = registered_assets["Storage"]
+    LOAD_NAMES = registered_assets.get("Load", [])
+    PV_NAMES = registered_assets.get("PV", [])
+    STORAGE_NAMES = registered_assets.get("Storage", [])
+    assert len(LOAD_NAMES + PV_NAMES + STORAGE_NAMES) != 0, (
+        "No assets are registered to the aggregator"
+    )
```

Comment:
This is not needed for the `register_asset_list` but only for the convenience of the user, right?

Reply:
Right! In particular, to deal with @fievelk 's concern expressed [here](https://github.com/gridsingularity/gsy-e-sdk/pull/221#discussion_r899879316).

Reply:
Oh, yes, you're right. The main concert is dealt with the `.get()` method. I introduced the assertion because I thought it would be no use to connect to a simulation with no assets on such lists. But, as you say, an Exception would be best. Do you have any in mind?